tags/Adcrowd/README.md

```diff
@@ -0,0 +1,10 @@
+# Adcrowd Google Tag Manager Template
+This is the template for Adcrowd Smartpixels in Google Tag Manager
+
```

Comment:
One last review request - could you update the README for Adcrowd to match the format used by other tags (same as how you did Pardot)?